asminc/vic20.inc

```diff
@@ -66,16 +66,43 @@ VIC_COLOR       := $900F        ; Border and background color
 ; ---------------------------------------------------------------------------
 ; I/O: 6522 VIA1
 
-VIA1            := $9110
-VIA1_JOY        := $9111
-VIA1_DDRB       := $9112
-VIA1_DDRA       := $9113
+VIA1            := $9110        ; *** Deprecated ***
```

Comment:
The `VIA1` and `VIA2` labels shouldn't be deprecated.  They can be used to refer to the entire chip.  
For example, they can be used in a loop to read/write many registers:
```asm
; Initiate the second VIA's registers.
        ldx #$0F - 1
@L1:    lda via2_data,x    
        sta VIA2,x
        dex
        bpl @L1
```

Reply:
Right, good point.